### PR TITLE
Fix ci

### PR DESCRIPTION
**Pull request by sam-github**

### CODE REVIEW

**Comment by sam-github** [comment]
test please

**Comment by rmg** [comment]
@slnode test please

**Comment by rmg** [comment]
@slnode test please

**Comment by rmg** [comment]
@slnode test please

File: test/cluster-store.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ var cluster = require('cluster');
 var expect = require('chai').expect;
 var ioServer = require('socket.io');
 var ioClient = require('socket.io-client');
-var debug = require('debug')('test');
+var debug = require('debug')('strong-cluster-socket.io-store:test');
```
Comment (bajtos) [inline test/cluster-store.js]: I prefer to use `test`, as it's much shorter to type. You are not likely to run tests from multiple modules, are you? Plus I find it useful to enable only logs from the production code, which requires even more typing with the debug schema you are proposing here.

File: test/cluster-store.js [hunk 2]
```diff
@@ -19,7 +19,6 @@ if (cluster.isWorker) {
 
 describe('clustered socket.io', function() {
   before(setupWorkers);
-  after(stopWorkers);
```
Comment (bajtos) [inline test/cluster-store.js]: I am wondering why this is no longer needed. Was it a workaround for a bug that has been fixed meanwhile?

File: test/cluster-store.js (hunk 1 above)
Comment (sam-github) [inline test/cluster-store.js]: As you wish. The scheme above is consistent with the rest of the strong-* repos, fwiw. I'm surprised you would run unit tests with DEBUG output, but no want the actual output from the unit tests, I usually do the opposite (want the top-level output from the test code, without the noise of all the underlying libraries, `DEBUG=*:test`). 

`DEBUG=*,-*:test` might work for the opposite.

But its no big deal either way.

File: test/cluster-store.js (hunk 2 above)
Comment (sam-github) [inline test/cluster-store.js]: See comment https://github.com/strongloop/strong-cluster-socket.io-store/commit/a46b0bbba1f884262638f4912ec0f05be1e505c7

File: test/cluster-store.js (hunk 1 above)
Comment (sam-github) [inline test/cluster-store.js]: reverted in 4d4e964

File: test/cluster-store.js (hunk 2 above)
Comment (sam-github) [inline test/cluster-store.js]: In short: I doubt it was ever needed, I suspect it was just an attempt to "cleanup", but the workers are always cleaned up on parent death.

File: test/cluster-store.js (hunk 1 above)
Comment (bajtos) [inline test/cluster-store.js]: Ah, I did not realise `DEBUG=*:test` works too.

> The scheme above is consistent with the rest of the strong-* repos

I see, in that case it's probably worth doing it your way. Feel free to rebase 4d4e964 away.

**Comment by bajtos** [comment]
LGTM. Please check why the builds are failing before landing.

**Comment by sam-github** [comment]
@slnode test please

**Pull request merged**